Design_Patterns/Singleton_and_Multithreading/Singleton_and_Multithreading.cpp

```diff
@@ -0,0 +1,103 @@
+#include <iostream>
+#include <list>
+#include <thread>
+#include <mutex>
+#include <atomic>
+#include <chrono>
+
+template <class T>
+class CustomList {
```

Comment:
not really a Singleton

Reply:
see commit 1a4b2fce6f366d858f11509eb63bd711699e9ed2 with the fix